wsproto/connection.py

```diff
@@ -275,6 +275,11 @@ def events(self):
                 if frame.opcode is Opcode.PING:
                     assert frame.frame_finished and frame.message_finished
                     self._outgoing += self._proto.pong(frame.payload)
+                    yield PingReceived(frame.payload)
+
+                elif frame.opcode is Opcode.PONG:
+                    assert frame.frame_finished and frame.message_finished
+                    yield PongReceived(frame.payload)
```

Comment:
Should we verify that the pong-payload is the same as what we sent earlier with `ping(payload)`?